src/internal_handlers/peer_management/mod.rs

```diff
@@ -248,8 +248,7 @@ pub fn handshake(
     listeners: &HashMap<SocketAddr, TransportType>,
     message_handlers: &MessageHandlers,
 ) -> Result<PeerId, PeerNetError> {
-    let mut bytes = (0 as u64).to_be_bytes().to_vec();
-    bytes.extend_from_slice(&PeerId::from_public_key(keypair.get_public_key()).to_bytes());
+    let mut bytes = PeerId::from_public_key(keypair.get_public_key()).to_bytes();
     //TODO: Add version in announce
     let listeners_announcement = Announcement::new(listeners.clone(), keypair).unwrap();
     bytes.extend_from_slice(&listeners_announcement.to_bytes());
```

Comment:
Above (can't comment more above) the offset should be 0 now that there is no id at front anymore

Reply:
Yes of course :facepalm: